nameko/cli/config.py

```diff
@@ -1,11 +0,0 @@
-from __future__ import print_function
```

Comment:
old file removed but new file not added?

Reply:
Whoops!